pokemongo_bot/__init__.py

```diff
@@ -92,6 +97,8 @@ def tick(self):
         self.check_session(self.position[0:2])
 
         for worker in self.workers:
+            if not self.config.navigator_move_to_forts and isinstance(worker(self), cell_workers.MoveToFortWorker):
```

Comment:
Is the goal of this to not use the MoveToFortWorker if we are using the path navigator, then this seems like it should be separate from the navigator and just be a flag that turns on the MoveToForts worker or not.